docs/cron/export.sh

```diff
@@ -1,4 +1,7 @@
 #!/bin/bash
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

Comment:
Below we set `ROOT` to the code root directory. So your script will be in `$ROOT/docs/cron` and I wonder whether this line is necessary.